gamemodes/terrortown/gamemode/client/cl_vskin/vgui/dform_ttt2.lua

```diff
@@ -268,9 +268,13 @@ function PANEL:MakeComboBox(data)
 			RunConsoleCommand(slf.m_strConVar, tostring(rawdata or value))
 		end
 
-		if isfunction(data.OnChange) then
-			data.OnChange(slf, index, value, rawdata)
-		end
+		-- run the callback function in the next frame since it takes
+		-- one frame to update the convar if one is set.
```

Comment:
oof that doesn't seem to be really reliable...

Reply:
Worked for me. Convars are updated in the next frame as it seems. There are many things that are handled like this in Gmod. So I don't see a problem with this.